apps/server/src/modules/board/controller/dto/board/content-element.url.params.ts

```diff
@@ -10,3 +10,13 @@ export class ContentElementUrlParams {
 	})
 	contentElementId!: string;
 }
+
+export class DrawingPermissionUrlParams {
+	@IsMongoId()
+	@ApiProperty({
+		description: 'The name of a drawing.',
+		required: true,
+		nullable: false,
+	})
+	drawingName!: string;
```

Comment:
If it is a mongoId, why not call the param "drawingId"?

Reply:
this is due to, old implementation naming, i will change that